modules/light-clients/08-wasm/types/config.go

```diff
@@ -0,0 +1,20 @@
+package types
+
+// contractMemoryLimit is the memory limit of each contract execution (in MiB)
+// constant value so all nodes run with the same limit.
+const ContractMemoryLimit = 32
+
+type WasmConfig struct {
```

Comment:
I added this type to pass to the Keeper constructor the parameters to instantiate the VM instead of passing each parameter individually. I thought that in case in the future we add more parameters, then we can extend this type, but do not break the API. I can add a `NewWasmConfig` function if deemed necessary. There is a [similar type used in `x/wasm`](https://github.com/CosmWasm/wasmd/blob/e5049ba686ab71164a01f6e71e54347710a1f740/x/wasm/types/types.go#L316), so I will move this to folder (in a follow-up PR), as Ethan suggested, to keep all the copied code in one place.

Reply:
would be nice to also get the `DefaultWasmConfig` that's used there.

Reply:
since this is in the `wasm/types` package, I think `Config` is a better name "wasmtypes.Config". However, I think we often refer to the import path as just `types` in which case WasmConfig is better. I will leave the choice to you. (my personal preference is use the `wasmtypes` alias, but am fine with either 👍 )

Reply:
I think I will leave it as is. :)

Reply:
Nice, I like this config struct idea. We should add some godoc to the top of struct also! :)

Reply:
When are we going to have chatGPT writing our godocs?